docs/conf.py

```diff
@@ -356,4 +356,13 @@
     ("py:mod", "aiohttp"),  # undocumented, no `.. currentmodule:: aiohttp` in docs
     ("py:class", "aiohttp.SimpleCookie"),  # undocumented
     ("py:class", "aiohttp.web.RequestHandler"),  # undocumented
+    ("py:meth", "aiohttp.ClientSession.request"),  # undocumented
+    ("py:class", "aiohttp.protocol.HttpVersion"),  # undocumented
+    ("py:class", "aiohttp.ClientRequest"),  # undocumented
+    ("py:class", "Payload"),  # undocumented
```

Comment:
This should be a full importable string, not just a class name.

Reply:
Do you mean full string "payload"?

Reply:
`package_name.module_name.ClassName`, just like others.

Reply:
Example:
```suggestion
    ("py:class", "aiohttp.payload.Payload"),  # undocumented
```